docker-custom/Dockerfile.custom

```diff
@@ -81,7 +81,7 @@ ENV NODE_RED_VERSION=$NODE_RED_VERSION \
 # ENV NODE_RED_ENABLE_SAFE_MODE=true    # Uncomment to enable safe start mode (flows not running)
 # ENV NODE_RED_ENABLE_PROJECTS=true     # Uncomment to enable projects option
 
-# User configuration directory volume
+# User configuration directory volume (if required)
```

Comment:
reasonable to leave the section here as this is an example Dockerfile 👍

Reply:
well - let's just remove them all.